### PR TITLE
Modified ticket-messages Endpoint with ClaimFiles

### PR DESCRIPTION
**Pull request by mschni2s**

### CODE REVIEW

**Comment by mschni2s** [comment]
fyi @aazon since hitmeister-web PR is approved, we'd like to send it live next week (prob. Tuesday or Wednesday), if this PR will be approved aswell. We will get in contact with PM respectively.

File: src/Namespaces/TicketMessagesNamespace.php [hunk 1]
```diff
@@ -20,15 +20,17 @@ class TicketMessagesNamespace extends AbstractNamespace
 	 * @param int    $ticketId
 	 * @param string $text
 	 * @param bool   $interimNotice
+     * @param array $claimMessageFiles
```
Comment (aazon) [inline src/Namespaces/TicketMessagesNamespace.php]:
```suggestion
     * @param Hitmeister\\Component\\Api\\Transfers\\MessageClaimFileTransfer[] $claimMessageFiles
```

Comment (aazon) [reply src/Namespaces/TicketMessagesNamespace.php]: Also, the `Hitmeister\\Component\\Api\\Transfers\\MessageClaimFileTransfer` transfer itself should be added to the repo

Comment (mschni2s) [reply src/Namespaces/TicketMessagesNamespace.php]: Should be there now

**Pull request merged**